app/services/proofing/aamva/logging_proofer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Proofing
+  module Aamva
+    class LoggingProofer < Proofing::Aamva::Proofer
```

Comment:
I am generally against subclasses like this. I would rather we add more boolean configs to the existing proofer, and remove the indirection from having a factory

Reply:
I guess, what I'm saying is, this class doesn't *add* anything significant that couldn't be done with a `log_results: true` option to the existing proofer

Reply:
@zachmargolis , true. Here we only logging for a small set of users potentially sensitive data. A little bit of over enginneering? The idea is to throw away the logging proofer and related tests after done feature testing, so dont have to change existing proofer's tests etc.

Also, this branch will not be merged to main branch as stated in the description.